saleor/core/utils/random_data.py

```diff
@@ -203,6 +204,7 @@ def create_product_variants(variants_data):
             continue
         defaults['product_id'] = product_id
         defaults['attributes'] = json.loads(defaults['attributes'])
+        defaults['price'] = Money(defaults['price']['amount'], settings.DEFAULT_CURRENCY)
```

Comment:
We have no `price` on the `ProductVariant`, instead we should handle `cost_price` and `price_override`. 
Both of them are nullable, so  we should handle it as well.
Something like:
```
if defaults['price_override']:
    defaults['price_override'] = Money(
        defaults['price_override'].amount, settings.DEFAULT_CURRENCY)
```
Same for the other field

Reply:
@Pacu2 thanks!